micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/MnProcessEngineConfiguration.java

```diff
@@ -84,7 +87,8 @@ public MnProcessEngineConfiguration(SynchronousTransactionManager<Connection> tr
                                         DataSource dataSource,
                                         MnArtifactFactory artifactFactory,
                                         MnBeansResolverFactory beansResolverFactory,
-                                        ProcessEngineConfigurationCustomizer processEngineConfigurationCustomizer) {
```

Comment:
switch the order so that plugins comes before processEngineConfigurationCustomizer